src/platform/lumin-runtime/platform-factory.js

```diff
@@ -67,6 +69,14 @@ export class PlatformFactory extends NativeFactory {
       throw new Error('PlatformFactory.createElement expects "name" to be string');
     }
 
+    if (name === 'scene') {
+      return this._createScene(name, container, ...args);
+    }
+
+    if (name === 'prism') {
+      return this._createPrism(name, container, ...args);
```

Comment:
why are you passing container if you don't actually use it?

Reply:
That will be refactored soon and I would like to preserve the interface. There won't be `_createScene`; `_createPrism` and `_createElement` but only `_create`. This refactoring will happen in different PR.

Reply:
https://github.com/magic-script/magic-script-components-lumin/issues/110